diff --git a/node_modules/node-robokassa/src/RobokassaHelper.js b/node_modules/node-robokassa/src/RobokassaHelper.js
index 05179ce..7506d26 100644
--- a/node_modules/node-robokassa/src/RobokassaHelper.js
+++ b/node_modules/node-robokassa/src/RobokassaHelper.js
@@ -164,7 +164,7 @@ class RobokassaHelper {
         break;
     }
 
-    // Validating and parsing request.
+    // Validating and parsing request t.
     let values = {};
     try {
       const keys = (Object.keys(options) || this.config.resultUrlKeys);
