diff --git a/node_modules/node-robokassa/src/RobokassaHelper.js b/node_modules/node-robokassa/src/RobokassaHelper.js
index 7c06537..b633da2 100644
--- a/node_modules/node-robokassa/src/RobokassaHelper.js
+++ b/node_modules/node-robokassa/src/RobokassaHelper.js
@@ -154,23 +154,24 @@ class RobokassaHelper {
     const method = (options.requestMethod || this.config.resultUrlRequestMethod);
 
     // Selecting request data object according to request method.
-    let data = {};
+    let data = options;
     switch (method.toUpperCase()) {
       case 'GET':
-        data = req.query;
+        data = options;
         break;
       case 'POST':
-        data = req.body;
+        data = options;
         break;
     }
+    console.log("wfwffw" + JSON.stringify(options))
 
     // Validating and parsing request.
     let values = {};
     try {
-      const keys = (options.keys || this.config.resultUrlKeys);
-      _.forEach(keys, (required, key) => {
+      const keys = (Object.keys(options) || this.config.resultUrlKeys);
+      _.forEach(keys, ( key) => {
         const value = data[key];
-        if (!value && required) {
+        if (!value) {
           throw new Error('Missing required key: ' + key);
         }
         if (value) {
@@ -179,7 +180,8 @@ class RobokassaHelper {
         }
       });
     } catch (error) {
-      res.status(400).send(error.message);
+      res.status = 400;
+      console.log(error)
       return;
     }
 
